pkg/client/client.go

```diff
@@ -1004,9 +1017,35 @@ func monitorKeepTerminatedEvent(
 					members,
 				)
 
-				tss.BroadcastRecoveryAddress(
+				if err != nil {
+					logger.Errorf(
+						"failed to retrieve member ids on keep [%s] termination: [%v]",
```

Comment:
We could rephrase the message to be consistent with other place we log failure of `AnnounceSignerPresence` :https://github.com/keep-network/keep-ecdsa/blob/9f5826bacf329ee119666cfe70eb62f947a4afcb/pkg/node/node.go#L202
```suggestion
						"failed to announce signer presence on keep [%s] termination: [%v]",
```